README.md

```diff
@@ -72,13 +77,44 @@ class PartyControllerTest < ActionController::TestCase
     with_variant your_ab_test_name: "B" do
       get :show
 
-      assert_rendered "new_show_template_to_be_tested"
-      assert_ab_test_rendered "example"
+      # Optional assertions about page content of the B variant
+    end
+  end
+end
+```
+
+Another common situation is to redirect one variant to another page:
+
+```ruby
+# test/controllers/party_controller_test.rb
+class PartyControllerTest < ActionController::TestCase
+  include GovukAbTesting::MinitestHelpers
+
+  should "redirect the user the B version" do
+    with_variant your_ab_test_name: "B", expected_redirect: { controller: "other_controller", action: "show" } do
```

Comment:
Yeah, I've been struggling with this API: if it handles all the request and response headers, it ends up knowing too much about the application. If it provides very low-level helpers like `assert_meta_tag` then there's a danger that people will forget to include one. You also end up having to pass the test and variant names around everywhere.

I'm not sure about `skip_asserts` because we only want to skip the assertion about the `meta` tag; the `Vary` assertion is still relevant for a redirect. Maybe the flag could be called something like `expect_meta_tag` or `assert_meta_tag` (with a default value of true)?